### PR TITLE
Feature: Add hide capability to buttons on sidebar (Settings > Appearance) and hamburger menu on sidebar for icons.

### LINKED ISSUE #65: Feature: Ability to hide "Add new service" and "Open workspace drawer" buttons from left sidebar
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

I have many added services and to open needed one I need to scroll vertically the list of services in left sidebar very often. But in the interface I see that the "Add new service" and "Open workspace drawer" buttons eats useful space, and I use them very-very rarely. "Disable notifications" button is used not so often too.

### Proposed Solution

Add the option to hide the "Add new service" and "Open workspace drawer", "Disable notifications" buttons from the left panel, to free up the space for Services icons.

### Alternatives Considered

Alternatively all those buttons, including the "Settings", can be hided via creating a single Hamburger menu button.

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by SpecialAro**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change
Feature: Add hide capability to buttons on sidebar (Settings > Appearance).
Feature: Add hamburger menu on sidebar for icons.

#### Motivation and Context
Feature: Add hide capability to buttons on sidebar (Settings > Appearance), solving issue https://github.com/ferdium/ferdium-app/issues/65.
Feature: Add hamburger menu on sidebar for icons following https://github.com/ferdium/ferdium-app/issues/65 suggestions.

#### Screenshots
image

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

#### Release Notes
Feature: Add hide capability to buttons on sidebar (Settings > Appearance)
Feature: Add hamburger menu on sidebar for icons.

### CODE REVIEW

**Comment by vraravam** [comment]
I thought you were putting these within a hamburger menu?

**Comment by SpecialAro** [comment]
You're right!
I'll implement it today so we can have both functionalities working 😁

**Comment by SpecialAro** [comment]
Added hamburger menu, as suggested
image
image

File: src/components/layout/Sidebar.js [hunk 1]
```diff
@@ -112,6 +115,10 @@ class Sidebar extends Component {
     this.setState({ tooltipEnabled: false });
   }
 
+  collapseMenu() {
+    this.setState({ isCollapsed: !this.state.isCollapsed });
```
Comment (santhosh-chinnasamy) [inline src/components/layout/Sidebar.js]: Hi @SpecialAro can we change this line to something like this
`this.setState(({ isCollapsed }) => ({ isCollapsed: !isCollapsed }));`

**Comment by MurzNN** [comment]
> Added hamburger menu, as suggested ![image](https://user-images.githubusercontent.com/37463445

Maybe a hamburger icon will be better than arrow here, to understand "what's this?" for users?

File: src/components/layout/Sidebar.js (hunk 1 above)
Comment (SpecialAro) [reply src/components/layout/Sidebar.js]: Will implement this on next commit! Thank you :D

**Comment by SpecialAro** [comment]
> Maybe a hamburger icon will be better than arrow here, to understand "what's this?" for users?

Agree with you. Changed the icon and the default to not collapsed.

**Comment by vraravam** [comment]
Testing this now on my local before the approval! Great job @SpecialAro !

**Comment by SpecialAro** [comment]
> Testing this now on my local before the approval! Great job @SpecialAro !

Thank you @vraravam ! And thank you everyone that helped me on this!

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
LGTM! Great work @SpecialAro !

**Pull request merged**